app.js

```diff
@@ -0,0 +1,59 @@
+// checking if local storage is empty
+if (localStorage.getItem('Added Books') === null) {
+  localStorage.setItem('Added Books', JSON.stringify([]));
+}
+// store book
+const storeData = JSON.parse(localStorage.getItem('Added Books'));
+function updateBook() {
+  localStorage.setItem('Added Books', JSON.stringify(storeData));
+}
+function createBooks(arr) {
+  let books = '';
+  for (let i = 0; i < arr.length; i += 1) {
+    books += `
+              <p>${arr[i].title}</p>
+              <p>${arr[i].author}</p>
+              <button onclick="removeBook(${i})">Remove</button>
+              <hr/>
+              `;
+  }
+  return books;
+}
+
+// Diplaying books
+function DispayAwesomeBooks() {
+  const listOfBooks = document.querySelector('.books-list');
+  listOfBooks.innerHTML = `
+                <ul class="book-list"/>
+                ${createBooks(storeData)}</ul>
+            `;
+}
+
+// Adding books
```

Comment:
- [ ] Your function of displaying books is not working properly,  after I add three books they are correctly rendered. But after I refresh the page the first book disappears and I am left with only 2 books, even though I didn't delete it. Also if I refresh again I am left with only one book. Check your function with regards to local storage and fix this issue